client/src/ui/molecules/language-menu/index.tsx

```diff
@@ -52,43 +55,55 @@ export function LanguageMenu({
               document.cookie = cookieValue;
             }
           }
+
+          ga("send", {
+            hitType: "event",
+            eventCategory: CATEGORY_LANGUAGE_TOGGLE,
+            eventAction: `Changing from the current locale: ${locale} to ${localeURL}`,
+            eventLabel: "change-language",
+          });
+
           navigate(localeURL);
+          window.scrollTo(0, 0);
         }
       }}
     >
-      <label htmlFor="select_language" className="visually-hidden">
-        Select your preferred language
-      </label>{" "}
-      <select
-        id="select_language"
-        name="language"
-        value={localeURL}
-        onChange={(event) => {
-          const { value } = event.target;
-          setLocaleURL(value);
-        }}
-      >
-        {/*
+      <fieldset>
```

Comment:
I think this needs to have the ID that we use as the anchor. E.g. 
```suggestion
      <fieldset id="select-language">
```

PS. Since this is something that would be seen in the browser (by the address bar) is it better to use hyphens instead of underscores since it looks less like raw code?